src/client/pages/Forever/index.tsx

```diff
@@ -61,25 +58,9 @@ export const Forever: React.FC<ForeverProps> = ({
   const textKeys = useTextKeys()
   const history = useHistory()
 
-  const { refetch: fetchCampaign } = useCampaignQuery({ skip: true })
```

Comment:
So it seems `useCampaignQuery` is not being used anymore and, therefore, we could remove it completely. That means get rid of `Campaign.graphql` file, righ?!

Reply:
Yep, will remember to clean that up later!